faith_presentation/src/commonMain/kotlin/net/thechance/mena/faith/presentation/feature/quran/bookmark/BookmarkViewModel.kt

```diff
@@ -25,24 +25,27 @@ import net.thechance.mena.faith.presentation.base.snackbar.SnackbarHandler
 class BookmarkViewModel(
     private val bookmarkRepository: BookmarkRepository,
     private val dispatcher: CoroutineDispatcher = Dispatchers.IO,
-    snackBarHandler: SnackbarHandler
+    snackBarHandler: SnackbarHandler,
 ) : BaseViewModel<BookMarkUiState, BookmarkEffect>(
-    BookMarkUiState(),
-    snackbarHandler = snackBarHandler
-), BookmarkInteractionListener {
-
-    private val cachedBookmarksFlow = createBookmarksPagingSource()
-        .map { pagingData -> pagingData.map { bookmark -> bookmark.toUiState() } }
-        .cachedIn(viewModelScope)
+        BookMarkUiState(),
+        snackBarHandler,
+    ),
+    BookmarkInteractionListener {
+    private val cachedBookmarksFlow =
+        createBookmarksPagingSource()
+            .map { pagingData ->
+                pagingData.map { bookmark -> bookmark.toUiState() }
```

Comment:
You can use the lambda shorthand with a function reference, like this:

```suggestion
                pagingData.map(AyahBookmark::toUiState)
```

Reply:
Great you noticed it, ok I will change it.